CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to embedchain
+
+Let us make contributing easy, collaborative and fun.
+
+## Submit your Contribution through PR
+
+To make a contribution, follow the following steps:
+
+1. Fork and clone this repository
+2. Do the changes on your fork with dedicated feature branch `feature/f1`
+3. If you modified the code (new feature or bug-fix), please add tests for it
+4. Include proper documentation / docstring and examples to run the feature
+5. Check the linting 
+6. Ensure that all tests pass 
+7. Submit a pull request
+
+For more details about pull requests, please read [GitHub's guides](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+
+### 📦 Package manager
+
+We use `poetry` as our package manager. You can install poetry by following the instructions [here](https://python-poetry.org/docs/#installation).
+
+Please DO NOT use pip or conda to install the dependencies. Instead, use poetry:
+
+```bash
+poetry install --all-extras
+or 
+poetry install --with dev
+
+#activate
+
+poetry shell
+```
+
+### 📌 Pre-commit
+
+To ensure our standards, make sure to install pre-commit before star to contribute.
+
+```bash
+pre-commit install
+```
+
+### 🧹 Linting
+
+We use `ruff` to lint our code. You can run the linter by running the following command:
+
+```bash
+ruff embedchain/
```

Comment:
Can you please change this to  `make lint`?

Reply:
done